src/AggregateFunctions/UniquesHashSet.h

```diff
@@ -108,7 +108,13 @@ class UniquesHashSet : private HashTableAllocatorWithStackMemory<(1ULL << UNIQUE
     inline size_t buf_size() const           { return 1ULL << size_degree; } /// NOLINT
     inline size_t max_fill() const           { return 1ULL << (size_degree - 1); } /// NOLINT
     inline size_t mask() const               { return buf_size() - 1; }
-    inline size_t place(HashValue x) const { return (x >> UNIQUES_HASH_BITS_FOR_SKIP) & mask(); }
+
+    inline size_t place(HashValue x) const {
```

Comment:
@SuzyWangIBMer you need to fix this - it fails style check
```suggestion
    inline size_t place(HashValue x) const
    {
```